content/spin/v2/testing-apps.md

```diff
@@ -147,13 +231,28 @@ build = "cargo component build --release"
 dir = "tests"
 ```
 
-The whole file will look like the following:
+## Configure App Storage
```

Comment:
An idea here is to have a H2 which is something like "Updating the App to Pass the Tests" with the "Update App Storage" and "Adding Busciness Logic" as subheads.  This groups the app-related tasks so the reader can think about them as a chunk rather than disconnected instructions.